xarray/core/datatree.py

```diff
@@ -1449,7 +1448,7 @@ def merge_child_nodes(self, *paths, new_path: T_Path) -> DataTree:
 
     # TODO some kind of .collapse() or .flatten() method to merge a subtree
 
-    def as_array(self) -> DataArray:
+    def as_dataarray(self) -> DataArray:
```

Comment:
Don't forget a quick update to `whats-new.rst` for this change.

Reply:
Great catch.